resolvers.py

```diff
@@ -0,0 +1,1979 @@
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+
+        pod_id = kwargs.get("pod_id")
+        project_name = kwargs.get("project_name")
+        account_name = kwargs.get("account_name")
+        billability = kwargs.get("billability")
+        duration = kwargs.get('duration').strip()
+        report_name = kwargs.get('report_name').strip().lower()
+        start_date = kwargs.get('start_date')
+        end_date = kwargs.get('end_date')
+
+        return self.cell_download_helper(pod_id, project_name, account_name, billability, duration, start_date, end_date, report_name)
+
+
+class OverallFreePoolByBandFilteredSummary():
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+        # filters variable takes band fields than needs to be shown
+        filters = kwargs.get("filters")
+        qs = []
+        if (len(filters) == 0):
+            qs = models.OverallBenchByBand.objects.all()
+        else:
+            qs_filtered = models.OverallBenchByBand.objects.filter(
+                band_name__in=filters)
+            qs_sum = qs_filtered.aggregate(bench=Sum('bench'), lop=Sum('lop'), resigned=Sum('resigned'), maternity=Sum('maternity'), allocation_pending=Sum('allocation_pending'), nominated=Sum('nominated'), non_deployable_total=Sum(
+                'non_deployable_total'), deployable_total=Sum('deployable_total'), sabbatical=Sum('sabbatical'), grand_total=Sum('grand_total'), grand_total_percentage=Sum('grand_total_percentage'))
+            qs_dummy = {"id": "Total", "band_name": 'Total'}
+            qs_dummy.update(qs_sum)
+            qs = list(chain(qs_filtered, [qs_dummy]))
+        return qs
+
+
+class OverallFreePoolByCapabilityFilteredSummary():
+    @custom_resolver_check_perm
+    def __call__(self, source, info, **kwargs):
+        # filters variable takes band fields than needs to be shown
+        filters = kwargs.get("filters")
+        qs = []
+        if (len(filters) == 0):
+            qs = models.OverallBenchByCapability.objects.all()
+        else:
+            qs_filtered = models.OverallBenchByCapability.objects.filter(
+                capability_name__in=filters)
+            qs_sum = qs_filtered.filter(band_name='Total').aggregate(bench=Sum('bench'), lop=Sum('lop'), resigned=Sum('resigned'), maternity=Sum('maternity'), allocation_pending=Sum('allocation_pending'), nominated=Sum(
+                'nominated'), non_deployable_total=Sum('non_deployable_total'), deployable_total=Sum('deployable_total'), sabbatical=Sum('sabbatical'), grand_total=Sum('grand_total'), grand_total_percentage=Sum('grand_total_percentage'))
+            qs_dummy = {"id": "Total",
+                        "capability_name": 'Total', "band_name": 'Total'}
+            qs_dummy.update(qs_sum)
+            qs = list(chain(qs_filtered, [qs_dummy]))
+        return qs
+
+
+class PeopleOnBenchHelper():
+
+    def __init__(self):
+        today = date.today()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Naive datetime objects are treated by many datetime methods as local times, We recommend that you create aware timestamps with the `now` or `fromtimestamp` functions using the `tzinfo` to attribute to represent times in UTC. Do not use the [datetime.utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) because it returns only a `naive` datetime object. 

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)